### DISCUSSION TITLE
[Feature] Private albums and hiding photos from the timeline

### DISCUSSION THREAD

**Discussion by gitdoc67** [immich-app/immich#5201] [Feature Request]
### The feature

For various reasons there could be sensitive photos in your library. This could be photos or screenshots of private information, or other types of sensitive photos. When I am browsing my photo library, I don't want these photos to be visible. I may have a friend or family member over my shoulder at the time. 

I would like to only see these in a private password protected view. I know I could theoretically accomplish this with a separate user account, but that would make the photo management and auto-import from iCloud or the Immich app more troublesome.

My proposal is the ability to create a private album which is password or PIN protected. Any photo that you add to a private album is then hidden when browsing your photo library/timeline. 

### Platform

- [ ] Server
- [x] Web
- [x] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2451
